.github/workflows/pip-compile-docs.yml

```
@@ -34,7 +34,7 @@ jobs:
       message: "ci: refresh docs build dependencies"
       base-branch: "${{ inputs.base-branch || 'devel' }}"
       pr-branch: "${{ inputs.pr-branch || 'pip-compile/devel/docs' }}"
-      nox-args: "-e 'pip-compile-3.10(requirements)' 'pip-compile-3.10(requirements-relaxed)'"
+      nox-args: "-e 'pip-compile-3.11(requirements)' 'pip-compile-3.11(requirements-relaxed)'"
```

@gotmax23 perhaps, this could be made an input under `workflow_dispatch:` above? Then, declare a global `env:` section providing a fallback for the value in relation to other events.

Also, let's get rid of that quote wrapping:
```suggestion
      nox-args: >-
        -e
        'pip-compile-3.11(requirements)'
        'pip-compile-3.11(requirements-relaxed)'
```

---

@gotmax23 should we commit the suggestion here from @webknjaz to get rid of the quote wrapping? looks like that's still valid to me. thanks.

---

Sure